mobileapp/components/shapes/Hat.tsx

```diff
@@ -0,0 +1,36 @@
+import { View } from "react-native";
+import Svg, { Path } from "react-native-svg";
+
+const Hat = () => {
+  return (
+    <View
```

Comment:
Can we found a better way to create these components ? There is a lot of repetition, maybe we can make a more generic component and pass props to it ?

Reply:
This should be replaced by icons later on if we can when we stop having loading issues with them.